M2/Macaulay2/m2/matrix.m2

```diff
@@ -215,6 +215,7 @@ protect Blocks
 blockMatrixForm=false;  -- governs expression Matrix inclusion of blocks
 expression Matrix := m -> (
     x := applyTable(entries m, expression);
+    if #x == 0 then x = {symbol zero => (target m, source m)};
```

Comment:
I don't understand why this line is not above the previous one (why go through all the entries if the matrix is zero anyway?)

Reply:
err, I mean, and why the test isn't simply `x==0`

Reply:
Great point!

Reply:
Ok, I've updated this so `expression` of a zero matrix always just uses the short form and never deals with the individual entries.